includes/gateway/class-omise-payment-installment.php

```diff
@@ -72,12 +72,14 @@ public function payment_fields()
 		$currency   = get_woocommerce_currency();
 		$cart_total = WC()->cart->total;
 		$capabilities = $this->backend->capabilities();
+		$installmentMinLimit = $capabilities->getInstallmentMinLimits();
```

Comment:
@aashishgurung why is the function `getInstallmentMinLimits` (plural) but the variable we save it to is `$installmentMinLimit` (singular)?